src/custom/components/CurrencyInputPanel/CurrencyInputPanelMod.tsx

```diff
@@ -82,8 +86,8 @@ export const InputPanel = styled.div<{ hideInput?: boolean }>`
 
 export const Container = styled.div<{ hideInput: boolean }>`
   border-radius: ${({ hideInput }) => (hideInput ? '8px' : '20px')};
-  border: 1px solid ${({ theme }) => theme.bg2};
-  background-color: ${({ theme }) => theme.bg1};
+  background: ${({ theme }) => theme.currencyInput?.background};
```

Comment:
this has an `undefined` or `falsy` background? is the browser set fallback fine?

Reply:
Likely would be good, but I think good practice to provide a fallback (in the end does rely on certain color variable to exist). Updated.